_docs/welcome/feature-stages.md

```diff
@@ -98,22 +84,4 @@ Below is our list of existing features and their current phases. This informatio
 | [Istio mTLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
 | [Pluggable Key/Cert Support for Istio CA](https://istio.io/docs/tasks/security/plugin-ca-cert.html)        | Beta
 
-### User Interface
-
-
-| Feature           | Phase       
-|-------------------|-------------------
-| [Grafana Dashboard](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html) | Alpha
-| [Service Graph](https://istio.io/docs/tasks/telemetry/servicegraph.html)    | Alpha
-
-
-### Performance and Scalability
```

Comment:
I think we want to clean this up rather than remove, should figure out how to word these.

Reply:
given we are alpha and we said performance is random for alpha and beta... it may generate more confusion at this point.